spyder/plugins/debugger/widgets/main_widget.py

```diff
@@ -341,6 +341,9 @@ def setup(self):
     def update_actions(self):
         """Update actions."""
         widget = self.current_widget()
+        if self.is_current_widget_empty():
+            return
+
```

Comment:
Maybe this validation could go before the `widget = self.current_widget()` line?